agent/proto/orchestrator.proto

```diff
@@ -315,7 +315,7 @@ message SumoLogicConfig {
 
 message HttpClientSettings {
     string url = 1;
-    repeated HttpHeader headers = 2;
```

Comment:
This is wrong, the http triggers use a struct like:
```
{
 key: string,
 value: string,
}
```

but Datastores use `map[string]string`